modules/publicize/class-jetpack-publicize-gutenberg.php

```diff
@@ -83,6 +83,22 @@ public function rest_get_publicize_connections_for_post( $request ) {
 		return wp_json_encode( $this->publicize->get_filtered_connection_data( $post_id ) );
 	}
 
+	/**
+	 * Retrieve full list of available Publicize connection services
+	 * send them as JSON encoded data.
+	 *
+	 * @see Publicize::get_available_service_data()
+	 *
+	 * @since 6.7.0
+	 *
+	 * @return string JSON encoded connection services data.
+	 */
+	public function rest_get_publicize_available_services() {
+		include_once( ABSPATH . 'wp-admin/includes/plugin.php' );
```

Comment:
What do we need this for?

Reply:
We need it for the `menu_page_url()` function that is being called in that method. It's defined in that file, and for admin ajax it works because it's an admin request and this file is automatically loaded.

Reply:
Let's add a comment (which method specifically) to make it easier to see why we need this.

Reply:
Added a comment. We also had another requirement (a submenu page to be present) - clarified that too. Let me know if it doesn't make sense.